acceptance-tests/src/main/java/it/com/atlassian/bitbucket/jenkins/internal/util/JenkinsUtils.java

```diff
@@ -0,0 +1,33 @@
+package it.com.atlassian.bitbucket.jenkins.internal.util;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+import static org.apache.commons.lang3.StringUtils.removeStart;
+
+public class JenkinsUtils {
+
+    private final String baseUrl;
+
+    private static final String CRUMB_ISSUER_URL = "/crumbIssuer/api/xml";
+
+    public JenkinsUtils(String baseUrl) {
+        this.baseUrl = baseUrl;
+    }
+
+    public RequestSpecification decorateWithCookie(RequestSpecification request) {
```

Comment:
I think we could we rename this method to also indicate that we are decorating with a Jenkins crumb too